install/deployment_yamls/k8s/meshery-service.yaml

```diff
@@ -15,6 +15,6 @@ spec:
     targetPort: 8080
   selector:
     io.kompose.service: meshery
-  type: NodePort
+  type: LoadBalancer
```

Comment:
@navendu-pottekkat I think lets stick with `NodePort` as we are not handling the scenario where in `LoadBalancer` type is not supported. Until we have that, lets stick to `Nodeport`.